homeassistant/components/light/opple.py

```diff
@@ -114,14 +121,24 @@ def turn_off(self, **kwargs):
 
     def update(self):
         """Synchronize state with light."""
+        prev_available = self.available
         self._device.update()
+
+        if prev_available == self.available and \
+                self._is_on == self._device.power_on and \
+                self._brightness == self._device.brightness and \
+                self._color_temp == self._device.color_temperature:
+            return
+
+        if not self.available:
+            _LOGGER.debug("Light %s is offline", self._device.ip)
```

Comment:
We could consider raising the logging level to warning, depending on how often we think this will occur.

Reply:
When people turn off their light through a wall switch, the light goes offline, so it's pretty common. So I think "debug level" is just fine.